packages/frontend-2/components/workspace/CreateDialog.vue

```diff
@@ -39,6 +39,10 @@ import { isRequired, isStringOfLength } from '~~/lib/common/helpers/validation'
 
 type FormValues = { name: string; description: string }
 
+const props = defineProps<{
+  navigateOnSuccess?: boolean
```

Comment:
I made it so we have an option to navigate to the workspace when its created. Defaults to false